### ISSUE TITLE
No release notes or upgrade guide for 1.0

### ISSUE THREAD

**Issue by KrayzeeKev** [axios/axios#5043]
#### Section/Content To Improve
Release notes
Upgrade guide

#### Suggested Improvement
The 1.0 Release in github includes a list of changes and inclusions, etc and then says "There are multiple deprecations, refactors and fixes provided in this release. Please read through the full release notes to see how this may impact your project and use case."

Where are the "full release notes".

Also, all upgrade documentation is talking about 0.19 or something. 

I presume there's an intention to provide the highlights of breaking changes we need to be aware of (if any - this could be a minor or null update from the last v0 release, it's unclear)

#### Relevant File(s)
[e.g. README.md]

**Comment by christhofer** [comment]
My project is still working after upgrading to v1, so I'm not complaining. YMMV of course.
I agree the release notes should include an upgrade guide or something like that.

**Comment by jasonsaayman** [comment]
If you read that statement you read the release notes. I am working on an upgrade guide.

**Comment by KrayzeeKev** [comment]
@jasonsaayman :-) No worries. Always enjoy a recursive reference.

**Comment by ldeveber** [comment]
Okay cool, it wasn't just me. Will it be updated with a link when it's ready? I'm hesitant to upgrade until I know what the deprecations are, but I don't want to stay out of date for too long :X

**Comment by jasonsaayman** [comment]
@KrayzeeKev sorry, I think that was a hectic day. I have these in the works, and will probably be up after this weekend. @ldeveber hopefully after the weekend.

**Comment by TheMoonDawg** [comment]
@jasonsaayman Was this ever completed? We're wanting to upgrade our project, but we don't see an upgrade guide anywhere, and it looks like the old outdated one was deleted.

**Comment by lornz** [comment]
Couldn't find an upgrade guide either. Stuck on 0.9.0 for that reason :(

**Comment by Shubham29012001** [comment]
Locally I shift my project axios version from 0.26.0 to 1.3.5 and everything works fine. When can we assume to get the migration guide or all the breaking changes?

**Comment by doutatsu** [comment]
Pretty wild there is still no migration guide and I am not willing to risk upgrading without knowing what changes are breaking...

**Comment by 54mu3l** [comment]
Crazy how such a widely used lib can be maintained so badly.

With 40 million downloads / week....
<img width="389" alt="image" src="https://github.com/axios/axios/assets/17022670/98dc3434-2aca-4939-870c-e79f493d5439">

**Comment by lukasamd** [comment]
I agree lack of info is bad but... it's open source so complaining about maintenance is... meh?

**Comment by kevingates** [comment]
Hi @jasonsaayman : was this upgrade guide ever completed?

**Comment by kmannislands** [comment]
>My project is still working after upgrading to v1, so I'm not complaining. YMMV of course.
I agree the release notes should include an upgrade guide or something like that.

> If you read that statement you read the release notes. I am working on an upgrade guide.

Then why a major version bump? To communicate that the `0.x` API has become stable?

I understand that this is an open source project that we all use at our own risk and contributors don't owe anyone anything however I think it's also fair for consumers of axios (there are likely millions) to assume that there would be breaking changes in `1.x` given [this is the documented convention that virtually all npm packages follow](https://docs.npmjs.com/about-semantic-versioning).

Whatever the reason for the version bump, a lot of people could be benefitted and a lot of time saved by writing it down.

I landed here after looking for the guide in an attempt to upgrade for [GHSA-wf5p-g6vw-rhxx](https://github.com/advisories/GHSA-wf5p-g6vw-rhxx) and landed here: https://github.com/axios/axios/blob/a52e4d9af51205959ef924f87bcf90c605e08a1e/MIGRATION_GUIDE.md which seems like the perfect place. 

From the activity chart, it seems that @jasonsaayman is taking a break from GitHub. Hopefully not from burnout dealing with tickets like this!

Perhaps another maintainer can weigh in here?

**Comment by kmannislands** [comment]
Looks like this ticket is a dupe of https://github.com/axios/axios/issues/5014#issuecomment-1498357937 which has a helpful link to a community-contributed guide here: https://github.com/bmuenzenmeyer/axios-1.0.0-migration-guide/

**Comment by jasonsaayman** [comment]
closing cause this is duplicated and yes @kmannislands was correct i was taking a step back for a bit but i am back with renewed energy and vision so will try get this in asap.

also thanks to the community for already creating some guides which i think work uite well.
